.php-cs-fixer.php

```diff
@@ -63,5 +63,7 @@
     ->setFinder(
         PhpCsFixer\Finder::create()
             ->in(__DIR__)
```

Comment:
```suggestion
            ->in(['src', 'tests'])
```
so no need "new" config after